testsuite/settings.py

```diff
@@ -1,7 +1,6 @@
 import os
 from datetime import datetime
 from vmck.utils import is_true, random_code
-from vmck.base_settings import *
```

Comment:
Leave this here, it was pulling the default settings.

You can ` # noqa: E123,E456` after this line.

Reply:
You can just say `noqa`, we understand that importing settings from a defaults file is a slightly-non-pythonic operation that is ok for this use case.

Reply:
Umm, you marked as resolved, but did not change the line :) At least write a message saying "i refuse to make this change" :p

Reply:
Umm, I readded the line with # noqa in my seccond commit. Maybe I don't understand what you want me to change?